src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -333,27 +333,26 @@ bool fx_muxer_t::resolve_hostpolicy_dir(host_mode_t mode,
     return false;
 }
 
-void fx_muxer_t::resolve_roll_forward(pal::string_t& fx_dir,
+fx_ver_t fx_muxer_t::resolve_most_compatible(const std::vector<fx_ver_t>& version_list,
```

Comment:
We should cotinue to call this resolve_roll_froward since that is what this function is doing.